### PR TITLE
Update permission FAQ for 13.3+

### PR DESCRIPTION
**Pull request by ParaDoX1994**
Adds information and changes regarding to permission granting on 13.3+ to clear confusion.

### CODE REVIEW

File: docs/ACCEL.md [hunk 1]
```diff
@@ -64,12 +64,14 @@ Currently Zoom relies partially on Metal and so needs a small binary patch. Dosd
 * [Zoom Non-Metal Fix](http://dosdude1.com/catalina/zoomnonmetal-new.command.zip)
 
 ## Unable to grant special permissions to apps (ie. Camera Access to Zoom)
+Currently for Ventura 13.3 and newer, due to new patches required, permissions are yet again broken.
```
Comment (dhinakg) [inline docs/ACCEL.md:67]: Keep the additional information that was formerly in the `0.2.4 and older Work-Around` sections

File: docs/ACCEL.md [hunk 2]
```diff
@@ -64,12 +64,14 @@ Currently Zoom relies partially on Metal and so needs a small binary patch. Dosd
 * [Zoom Non-Metal Fix](http://dosdude1.com/catalina/zoomnonmetal-new.command.zip)
 
 ## Unable to grant special permissions to apps (ie. Camera Access to Zoom)
+Currently for Ventura 13.3 and newer, due to new patches required, permissions are yet again broken. 
+There are two ways around it, you can either use tccplus to grant permissions (found in the workaround dropdown) or try the new [AMFIPass beta build.](https://github.com/dortania/OpenCore-Legacy-Patcher/releases/tag/amfipass-beta-test)
```
Comment (dhinakg) [inline docs/ACCEL.md:68]: Do not link to the AMFIPass beta

**Review by dhinakg — CHANGES_REQUESTED** [review]

File: docs/ACCEL.md (hunk 1 above)
Comment (ParaDoX1994) [reply docs/ACCEL.md:67]: Done 👍

File: docs/ACCEL.md (hunk 2 above)
Comment (ParaDoX1994) [reply docs/ACCEL.md]: Done 👍

**Pull request merged**